lib/runner.js

```diff
@@ -50,12 +50,19 @@ function repositoryToNativePackageName(repository/*:string*/) {
   //
   // matches[1] = "rtfeldman"
   // matches[2] = "node-test-runner"
-  var matches = repository.match(/\/([^\/]+)\/([^\/\.]+)\.git/);
+  var matches = repository.match(/\/([^\/]+)\/([^\/]+)\.git/);
 
   if (matches instanceof Array) {
     var username = matches[1];
     var repoName = matches[2];
 
+    //"A dot in repository name breaks `elm-make`" https://github.com/elm-lang/elm-make/issues/106
+    if (repoName.indexOf('.') >= 0) {
```

Comment:
While we're here, might as well check the username. Even though github doesn't allow periods in usernames, there is no requirement that the repo _actually exists_, so this can happen.